Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/LinePointer.cs

```diff
@@ -140,10 +140,11 @@ public override void OnPostSceneQuery()
         {
             base.OnPostSceneQuery();
 
-            LineBase.enabled = IsInteractionEnabled;
-            BaseCursor?.SetVisibility(IsInteractionEnabled);
+            bool isEnabled = IsInteractionEnabled;
```

Comment:
I'm not sure what these changes accomplish. What am I missing?

Reply:
IsInteractionEnabled isn’t a variable, it’s basically a function. Each call is really testing many variables and levels of inheritance. It’s minor but seemed wasted overhead to do like I believe 5-8 Boolean conditions repeatedly instead of just cashing the results.